modules/auth/saas/settings/settings.tsx

```diff
@@ -27,9 +31,12 @@ const Settings = () => {
     resolver: yupResolver(settings_schema(t)),
   });
 
-  const [showPassword, setShowPassword] = useState(false);
-  const [showConfirmPassword, setShowConfirmPassword] = useState(false);
-
+  const handleRevokeOAuth = async (): Promise<void> => {
+    const response = await Service.require("oauth").revokeOauth();
```

Comment:
IF we have 3 oauth, which one is revoking? I think we require to pass the provider name (same as we do on auth)